www/includes/easyparliament/templates/html/mp/divisions.php

```diff
@@ -86,9 +61,11 @@
                             <?php if ( isset($policy['position']) ) { ?>
                                 <div class="panel">
                                     <h3 class="policy-vote-overall-stance">
-                                        <?= $full_name ?> <?= $policy['position']['voted'] == 'never voted' ? $policy['position']['voted'] . ' on' : 'voted ' .$policy['position']['voted'] ?> <?= $policy['desc'] ?>
+                                        <?= $full_name . ' ' . $policy['position']['full_description'] . ' ' . $policy['desc'] ?>
                                     </h3>
 
+                                    <p>TheyWorkForYou has automatically calculated this MP&rsquo;s stance based on all of their votes on the topic. <a href="http://www.publicwhip.org.uk/mp.php?mpid=<?= $member_id ?>&dmp=<?= $policy['policy_id'] ?>">You can browse the source data on PublicWhip.org.uk</a>.</p>
```

Comment:
And could this be line-wrapped, so it isn't more luck that I spot that ampersand? :)